cypress/e2e/po/pages/explorer/projects-namespaces.po.ts

```diff
@@ -29,7 +30,63 @@ export default class ProjectNamespacePagePo extends PagePo {
     return this.createProjectNamespaceButton().click();
   }
 
+  createProjectButton() {
```

Comment:
This should be accessed via `ResourceListMastheadPo` (as that's where the button lives)

```
 masthead() {
   return new ResourceListMastheadPo(this.self());
}

```